### ISSUE TITLE
Template erro causing 500 errors even when Harvest Plugin is deactivated

### ISSUE THREAD

**Issue by newism** [mully/redmine_harvest#1]
Hi Bill,

I am having some issues with the Harvest plugin. It was working fine and then I restarted my server. 
The error logs show the following: http://pastie.org/721192

Any insight would be great.

I'm running:

Rails 2.3.4
Harvester 0.0.1
Passenger 2.2.7
Redmine 0.8.7.stable

Kind regards

**Comment by newism** [comment]
I'm also getting this template rendering issue on the Hrvest module tab: http://pastie.org/721218

**Comment by mully** [comment]
Strange that it would stop working after a server restart. I am unable to replicate this issue after several restarts.

The "l_hours" method is on line 29 of lib/redmine/i18n.rb and included on line 22 of application_controller.rb.  Not sure why this would be undefined. 

Let me know if you stumble across any other clues.

**Comment by newism** [comment]
Ok looks like we are puling from the 0.8-stable branch of Github. Unfortunatley the lib/redmine/i18n.rb file has not been added to the branch:

http://github.com/edavis10/redmine/tree/0.8-stable/lib/redmine/

**Comment by newism** [comment]
The version for the 0.8-stable branch is clearly marked: http://github.com/edavis10/redmine/blob/master/lib/redmine/version.rb

**Comment by mully** [comment]
Good catch on the branch.  I noticed that file does not exist on the 0.8.7 tag either which I thought I was using, but apparently that is not the case.

So it looks like it may work if you substituted this

<pre><code>l_hours(harvest_time)</pre></code>

with this

<pre><code>lwr(:label_f_hour, harvest_time)</pre></code>

Any idea how this could have worked for you prior to restarting your server?

**Comment by newism** [comment]
> Any idea how this could have worked for you prior to restarting your server?

Well I think I may have been switching between my local machine and the actual server. There is some weirdness going on but I'm not going to try and debug it until I can get a stable version of Redmine. 

Which branch / version are you running inhouse?

**Comment by newism** [comment]
Hey Mully,

I just downloaded the official redmine-0.8.7.tar.gz release from http://rubyforge.org/frs/?group_id=1850 and here is still no lib/redmine/i18n.rb

Any ideas? I really would like to get your integration working.

Cheers
